nn/linear.go

```diff
@@ -26,7 +26,7 @@ func Linear(in, out int64, bias bool) *LinearModule {
 	}
 	l.Weight = torch.Empty([]int64{out, in}, true)
 	if bias {
-		l.Bias = torch.Empty([]int64{out, 1}, true)
+		l.Bias = torch.Empty([]int64{out}, true)
```

Comment:
This the change on this line affect random initialization?

Reply:
No, the bias shape should be `{out}`, it's a  bug, c.f. https://github.com/pytorch/pytorch/blob/v1.6.0/torch/csrc/api/src/nn/modules/linear.cpp#L36